it/test/repositories/MongoJourneyAnswersRepositoryISpec.scala

```diff
@@ -98,12 +102,12 @@ class MongoJourneyAnswersRepositoryISpec extends MongoSpec with DefaultPlayMongo
 
     "update already existing answers (values, updateAt)" in {
       val result = for {
-        _ <- repository.upsertAnswers(ctx, Json.obj("field" -> "value"))
-        _ <- repository.upsertAnswers(ctx, Json.obj("field" -> "updated"))
+        _       <- repository.upsertAnswers(ctx, Json.obj("field" -> "value"))
+        _       <- repository.upsertAnswers(ctx, Json.obj("field" -> "updated"))
         updated <- find(filters)
       } yield updated.headOption.value
 
-      val expectedExpireAt = ExpireAtCalculator.calculateExpireAt(now)
+      val expectedExpireAt = now.atZone(ZoneOffset.UTC).plusDays(mockAppConfig.timeToLive).truncatedTo(ChronoUnit.DAYS).toInstant
```

Comment:
Maybe we could add a test case where we test the mongo deletion? (it might take 30 seconds even to test a 1 second TTL - for test, so it might increase the testing time by 30 seconds though :/)

Reply:
We have the Unit set to DAYS in the implementation so unable to test 30 seconds

```
IndexOptions()
            .expireAfter(appConfig.timeToLive, TimeUnit.DAYS)
            .name("expire_at")
        )
```

Reply:
Agreed with @modivilioglu that it will be good to refactor and parametrize this later